util/execdetails/execdetails.go

```diff
@@ -149,7 +149,7 @@ func (d ExecDetails) String() string {
 			parts = append(parts, CommitBackoffTimeStr+": "+strconv.FormatFloat(time.Duration(commitBackoffTime).Seconds(), 'f', -1, 64))
 		}
 		if len(commitDetails.Mu.BackoffTypes) > 0 {
-			parts = append(parts, BackoffTypesStr+": "+fmt.Sprintf("%v", commitDetails.Mu.BackoffTypes))
+			parts = append(parts, BackoffTypesStr+": "+strings.Replace(fmt.Sprintf("%v", commitDetails.Mu.BackoffTypes), " ", ",", -1))
```

Comment:
```
Backoff_types: [tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss,tikvRPC,regionMiss]
```

Actually, there are only 2 kinds of backoff: `regionMiss` and `regionMiss`, there are too much duplicate information. So I want to change the output to `backoff_type*count`:

```sql
Backoff_types: [tikvRPC*10,regionMiss*10]
``` 

And if the count is 1, just ignore it, such as:

```sql
Backoff_types: [tikvRPC*10,regionMiss*10,pdRPC]
``` 

@bb7133 @breeswish what do you think?

Reply:
ping~ @bb7133 @qw4990 @breeswish PTAL

Reply:
This collapse looks great!

Reply:
Does the order matters?

Reply:
Maybe order by the count with desc is easier to read.

Reply:
@sylzd Please resolve this comment~

Reply:
It does look Great, but what i worried is that the compatibility maybe broken more, which zoom in the radius of damage.
like this line:
https://github.com/pingcap/tidb/blob/e5bf177cd91ad689176c5f647e1e37d5cd47899c/util/stmtsummary/statement_summary.go#L763

Reply:
why this change will affect this?

Reply:
Aha~ My mistake, `BackoffTypes` here seems got from stats in memory instead of slow.log.  this case is not suitable, what I mean is that some tools like `pt-query-digest` or some application on table `slow_query` may be a little confused~  

BTW, I am not very sure, what's the purpose of the backoffTypes shown in slow log. 🤣